src/math/base.rs

```diff
@@ -39,12 +39,20 @@ impl<S: RealField + Bounded> IntersectAabb<S> for CachedAxesIntersector<S> {
 /// Use this macro as a crutch for the missing
 /// `impl<'a, S, T: ConvexPolyhedron<S>> HasAabbIntersector<'a, S> for T`.
 macro_rules! has_aabb_intersector_for_convex_polyhedron {
-    ($type:ty) => {
-        impl<'a, S: RealField> HasAabbIntersector<'a, S> for $type {
+    ($type:tt<$param:tt>) => {
+        impl<'a, S: RealField> HasAabbIntersector<'a, S> for $type<$param> {
             type Intersector = CachedAxesIntersector<S>;
             fn aabb_intersector(&'a self) -> Self::Intersector {
                 self.intersector().cache_separating_axes_for_aabb()
             }
         }
     };
+    ($type:ty) => {
+        impl<'a> HasAabbIntersector<'a, f64> for $type {
+            type Intersector = CachedAxesIntersector<f64>;
+            fn aabb_intersector(&'a self) -> Self::Intersector {
+                self.intersector().cache_separating_axes_for_aabb()
+            }
+        }
+    };
```

Comment:
If we implemented `WebMercatorRect` for all float types, we wouldn't need this change, right? I think this would make the code a little cleaner....

Reply:
Yes.
Aside: In the long term, do you think having these be generic makes sense, or should we go towards making everything f64?

Reply:
I think generics don't hurt here so much. Users can still decide on precision themselves... I would keep generic math stuff as generic as possible.

Reply:
As discussed in person, `f64` makes more sense.